src/test/suite/_completion/completion.java.test.ts

```diff
@@ -65,11 +65,22 @@ async function testCompletion(
 	let javaExtension: vscode.Extension<any> | undefined;
 	await waitUntil(() => {
 		javaExtension = vscode.extensions.getExtension('redhat.java');
-		return javaExtension?.isActive && javaExtension?.exports?.status === "Started";
+		const isJavaExtensionActive = javaExtension?.isActive;
+		if(isJavaExtensionActive) {
+			try {
+				return javaExtension?.exports?.status === "Started";
+			} catch (error) {
+				console.log(error);
+				return false;
+			}
+		} else {
```

Comment:
the else case is not really required. a simple `return false;` would be sufficient. wdyt?

Reply:
I think it is more readable to have clearly the if/else